README.md

```diff
@@ -1,10 +1,14 @@
-firesim-software
+Fire Marshal (firesim-software)
```

Comment:
I propose we reduce this readme to a few lines that aggressively redirects to FireSim's docs.

Reply:
Agreed. Fixed.